src/seedu/addressbook/data/person/Person.java

```diff
@@ -84,4 +84,15 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
```

Comment:
call the method elsewhere in addressbook so that you can actually see the output!